package.json

```diff
@@ -24,8 +25,21 @@
     "@slack/web-api": "^6.7.2",
     "axios": "^0.27.2",
     "jsonwebtoken": "^9.0.0",
+    "node-cache": "^5.1.2",
     "winston": "^3.8.2"
   },
+  "peerDependencies": {
+    "ioredis": "^5.2.5",
+    "redis": "^4.5.1"
```

Comment:
if u can use lower versions it would be nice for our customers

Reply:
done